src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2762,10 +2762,6 @@ private void EnterParameters()
                 }
             }
 
-            // The partial definition part may include optional parameters whose default values we want to simulate assigning at the beginning of the method
-            // https://github.com/dotnet/roslyn/issues/73772: is this actually used/meaningful?
-            methodSymbol = methodSymbol.PartialDefinitionPart ?? methodSymbol;
```

Comment:
This and the other deleted usage of partial APIs in nullable are no longer necessary per https://github.com/dotnet/csharplang/blob/main/proposals/nullable-parameter-default-value-analysis.md#suggested-change-to-declaration-analysis.